packages/react-composites/tests/app/callwithchat/BaseApp.tsx

```diff
@@ -17,7 +17,7 @@ export function BaseApp(props: { queryArgs: CommonQueryArgs; adapter?: CallWithC
     return <h3>Initializing call-with-chat adapter...</h3>;
   }
   return (
-    <div style={{ position: 'fixed', width: '100%', height: '100%' }}>
+    <div id="test-app-root" style={{ position: 'fixed', width: '100%', height: '100%' }}>
```

Comment:
_slight_ misnomer as the hermetic app root actually has hidden composites as well as this base app
```suggestion
    <div id="base-app-root" style={{ position: 'fixed', width: '100%', height: '100%' }}>
```
?

Reply:
Still a misnomer then. What I want to say is that this be the app you're trying to test.

The HermeticApp / BaseApp split is implementation detail.

Perhaps `app-under-test-root` captures it best?

Reply:
Ahh I think just leave it as is then